src/Components/__stories__/Popover.story.tsx

```diff
@@ -0,0 +1,113 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+
+import styled from "styled-components"
+import Button from "../Buttons/Default"
+import Input from "../Input"
+import { Popover } from "../Popover"
+
+const Container = styled.div`
+  padding: 50px;
+  height: 1400px;
```

Comment:
Why does this container have to have a very long height?

Reply:
I was testing scrolling.